src/localCollection.js

```diff
@@ -91,10 +88,8 @@ class LocalCollection extends EventEmitter {
    */
   _onRemoved(id) {
     var doc = this._docs[id];
-    if (!doc) {
-      if (this._supressRemovalWarnings) return;
```

Comment:
You should be able to remove the `_supressRemovalWarnings` property and option in the constructor too.